src/global.d.ts

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 New Vector Ltd.
+Copyright 2023  New Vector Ltd
```

Comment:
This looks wrong?

Reply:
Woops, I actually re-created that file and copy-pasted the copyright from the wrong place. Fixed